lib/workers/repository/process/extract-update.ts

```diff
@@ -142,7 +143,10 @@ export async function extract(
   const cachedExtract = cache.scan[baseBranch!];
   const configHash = fingerprint(generateFingerprintConfig(config));
   // istanbul ignore if
-  if (isCacheExtractValid(baseBranchSha!, configHash, cachedExtract)) {
+  if (
+    overwriteCache &&
+    isCacheExtractValid(baseBranchSha!, configHash, cachedExtract)
```

Comment:
So if overwriteCache=false then also don't _read_ the cache either?

BTW shouldn't we be trying to cache the reconfigure branch too though?

Reply:
caching seems to make sense 🤔

Reply:
We do use caching for the reconfigure Pr. If the branchSha of reconfigre branch hasn't changed we do not process it. 

> So if overwriteCache=false then also don't read the cache either?

Yes, both. For reconfigure pr we do not have intelligent caching like we have for normal extraction using fingerprints. 
A simple branchSha comparison is used to prevent the reconfigure branch processing.

Reply:
That seems ok then